srcpkgs/empathy/template

```diff
@@ -1,27 +1,39 @@
 # Template file for 'empathy'
 pkgname=empathy
-version=3.12.14
+version=3.25.90
 revision=1
-disable_parallel_build=yes
 build_style=gnu-configure
 configure_args="
  --enable-gst-1.0=yes --enable-gudev=yes --enable-spell=yes
  --disable-schemas-compile --disable-static --enable-goa=no
  --enable-ubuntu-online-accounts=no --enable-geocode=yes
  --enable-location=yes --enable-nautilus-sendto=no --enable-map=yes"
 hostmakedepends="pkg-config intltool gnome-doc-utils itstool"
-makedepends="
- libcanberra-devel clutter-gst-devel clutter-gtk-devel
+makedepends="libcanberra-devel clutter-gst-devel clutter-gtk-devel
  telepathy-glib-devel telepathy-logger-devel libnotify-devel
  telepathy-farstream-devel telepathy-mission-control-devel
  evolution-data-server-devel geocode-glib-devel folks-devel
  webkit2gtk-devel libgudev-devel pulseaudio-devel cheese-devel
  enchant-devel NetworkManager-devel libchamplain-devel
- gnutls-devel gsettings-desktop-schemas-devel geoclue2-devel iso-codes"
+ gnutls-devel gsettings-desktop-schemas-devel geoclue2-devel
+ iso-codes"
 depends="gsettings-desktop-schemas>=3.12 iso-codes"
 short_desc="GNOME instant messaging client using the Telepathy framework"
 maintainer="Orphaned <orphan@voidlinux.org>"
+license="GPL-2.0-only"
 homepage="http://live.gnome.org/Empathy"
-license="GPL-2"
 distfiles="${GNOME_SITE}/$pkgname/${version%.*}/$pkgname-$version.tar.xz"
-checksum=7d86942ce97edd10ade0e6ae6a210d35e4d627fe4d223377d71fd1840bc6e3a3
+checksum="cc9289d0180e23e566fb4972562d9ebd2bba2f457d9fb954f6d4b0ea9903c16a"
+disable_parallel_build=yes
+
+if [ "${CROSS_BUILD}" ]; then
+	hostmakedepends+=" telepathy-mission-control-devel glib-devel geoclue2-devel"
+	_commit=39f04a882154e82d43f211d18954401bb561fe12
+	distfiles+=" https://gitlab.gnome.org/GNOME/telepathy-account-widgets/-/archive/master/$_commit.tar.bz2?path=tp-account-widgets>taw-tpaw-$_commit.tar.bz2"
+	checksum+=" ac56d16592f5533fdf6a58be5536e34b319d4ecd48759653f019d48e20db24f6"
+
+	pre_build() {
+		mv ../telepathy-account-widgets-master-$_commit-tp-account-widgets/* .
+		rmdir ../telepathy-account-widgets-master-$_commit-tp-account-widgets
+	}
+fi
```

Comment:
i hope you don't mind i fixed this mess in #17564

Reply:
Thanks!